src/app/service-file.service.ts

```diff
@@ -1,27 +1,27 @@
 import { Injectable } from '@angular/core';
-import { Observable } from 'rxjs';
-import { HttpClient, HttpParams } from '@angular/common/http';
+import {  Observable } from 'rxjs';
+import { HttpClient } from '@angular/common/http';
+import { environment } from 'src/environments/environment';
 
 @Injectable({
   providedIn: 'root'
 })
 export class ServiceFileService {
-  url: string = 'http://localhost:3000/items'
+  url: string = environment.apiUrl
  
   constructor(private http: HttpClient) { }
   getPosts() {
     return this.http.get(this.url)
   }
-
-  getUsers() {
-    //return this.http.get("https://run.mocky.io/v3/1e45e371-a0c9-4b53-92b9-53aaec34b063");
-    return this.http.get("http://localhost:3000/users/");
-  }
-  deleteRequest(id: number) {
+  deleteRequest(id: any): Observable<any> {
     return this.http.delete(`${this.url}/${id}`);
   }
 
-  postRequest(post: any) {
+  postRequest(post: any): Observable<any> {
     return this.http.post(this.url,post)
   }
+
+  putRequest(id: any, data: any): Observable<any> {
```

Comment:
Don't user any as type on overall application

Reply:
Provided a type to all the variables instead of type "any".